packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -86,13 +91,15 @@ export async function prefetchLatestSnapshot(
 			storageToken,
 			odspLogger,
 			snapshotOptions,
-			snapshotFormatFetchType,
+			undefined,
```

Comment:
Should we remove this argument from the list of arguments to prefetchLatestSnapshot() ?
That's breaking change, so maybe that should be done in next.
In general, should we remove from code ability to dictate what format to fetch? It feels like we have been in prod enough to say that we will always ask for binary, with fallback to json, and thus any code that provides the opposite should be cleaned.
I'd suggest doing it as separate PR, and across all code paths.

Reply:
Yes, I was thinking of doing it in the next branch.